spring-rabbit/src/main/java/org/springframework/amqp/rabbit/support/micrometer/RabbitListenerObservation.java

```diff
@@ -72,4 +73,28 @@ public String asString() {
 
 	}
 
+	/**
+	 * Default {@link RabbitListenerObservationConvention} for Rabbit listener key values.
+	 */
+	public static class DefaultRabbitListenerObservationConvention implements RabbitListenerObservationConvention {
+
+		/**
+		 * A singleton instance of the convention.
+		 */
+		public static final DefaultRabbitListenerObservationConvention INSTANCE =
+				new DefaultRabbitListenerObservationConvention();
+
+		@Override
+		public KeyValues getLowCardinalityKeyValues(RabbitMessageReceiverContext context) {
+			return KeyValues.of(RabbitListenerObservation.ListenerLowCardinalityTags.LISTENER_ID.asString(),
```

Comment:
FYI: I have fixed this tangle just using literals instead of enums: https://github.com/spring-projects/spring-integration/pull/3896/files#diff-1966bee1c00ff5bd693214849eb988472f9f86f986023f598cc8d344a7740afbR39